data/cameras.xml

```diff
@@ -7930,6 +7930,17 @@
 		<Crop x="0" y="0" width="0" height="0"/>
 		<Sensor black="512" white="16300"/>
 	</Camera>
+	<Camera make="SONY" model="ILCE-7M4">
+		<ID make="Sony" model="ILCE-7M4">Sony ILCE-7M4</ID>
+		<CFA width="2" height="2">
+			<Color x="0" y="0">RED</Color>
+			<Color x="1" y="0">GREEN</Color>
+			<Color x="0" y="1">GREEN</Color>
+			<Color x="1" y="1">BLUE</Color>
+		</CFA>
+		<Crop x="0" y="0" width="0" height="0"/>
```

Comment:
There's definitely a need to remove some pixels on the right side, the question is - how many exactly?